hconfig/mapl3hconfig_get.F90

```diff
@@ -0,0 +1,39 @@
+#include "MAPL_ErrLog.h"
+module mapl3hconfig_get
+
+   use mapl3hconfig_get_private
+   use mapl_ErrorHandling
+   use mapl_KeywordEnforcer
+
+   implicit none
+
+   private
+
+   public :: MAPL_HConfigGet
+
+   interface MAPL_HConfigGet
+      module procedure :: hconfig_get_scalar
+   end interface MAPL_HConfigGet
+
+contains
+
+   subroutine hconfig_get_scalar(hconfig, keystring, value, message, unusable, value_is_set, rc)
+      type(ESMF_HConfig), intent(inout) :: hconfig
+      character(len=*), intent(in) :: keystring
+      class(*), intent(inout) :: value
+      character(len=:), allocatable, intent(inout) :: message
+      class(KeywordEnforcer), optional, intent(in) :: unusable
+      logical, optional, intent(out) :: value_is_set
+      integer, optional, intent(out) :: rc
+      logical :: found
+
+      _UNUSED_DUMMY(unusable)
+
```

Comment:
Weiyuan suggests that these macros be moved to bottom of procedures.  I agree.

Reply:
OK. I have no opinion. That's fine.

Reply:
Done.